packages/e2e-tests/pages/patients/PatientDetailsPage/PatientDetailsPage.ts

```diff
@@ -260,9 +284,42 @@ export class PatientDetailsPage extends BasePatientPage {
     return this.notesPane;
   }
 
+  async navigateToVitalsTab(): Promise<void> {
+    await this.vitalsTab.click();
+  }
+
+  async navigateToImagingRequestTab(): Promise<void> {
+    await this.encountersList.first().waitFor({ state: 'visible' });
+    await this.encountersList.first().click();
+    await this.imagingTab.click();
+  }
+
+  async navigateToPatientDetailsTab(): Promise<PatientDetailsTabPage> {
+    await this.patientDetailsTab.click();
+    if (!this.patientDetailsTabPage) {
+      this.patientDetailsTabPage = new PatientDetailsTabPage(this.page);
+    }
+    return this.patientDetailsTabPage;
+  }
+
+  async navigateToAllPatientsPage(): Promise<AllPatientsPage> {
+    await this.page.goto(constructFacilityUrl(`${routes.patients.all}`));
+    return new AllPatientsPage(this.page);
+  }
+
+  async navigateToMedicationTab(): Promise<EncounterMedicationPane> {
+    await this.encountersList.first().waitFor({ state: 'visible' });
+    await this.encountersList.first().click();
+    await this.encounterMedicationTab.click();
+    if (!this.encounterMedicationPane) {
+      this.encounterMedicationPane = new EncounterMedicationPane(this.page);
+    }
+    await this.encounterMedicationPane.waitForPaneToLoad();
+    return this.encounterMedicationPane;
+  }
 
   async goToPatient(patient: Patient) {
-    await this.page.goto(constructFacilityUrl(`/#/patients/all/${patient.id}`));
+    await this.page.goto(constructFacilityUrl(`/patients/all/${patient.id}`));
```

Comment:
@sepidehdehghani992 I think this was the key change. Routes in tamanu web no longer start with `#`. There was an update last week that changed that.